### ISSUE TITLE
select filaments 11-16 by number

### ISSUE THREAD

**Issue by UBaer21** [bambulab/BambuStudio#5346]
**Is your feature request related to a problem? Please describe.**
I can select filament 10 by pressing 1 then 0,  doesn't work for 11 ...

**Describe the solution you'd like**
select 11-16  same as i can select 10 by pressing the numbers

**Describe alternatives you've considered**
select by mouse dropdown

**Comment by JohnSmithCYM** [comment]
![20241126163008_rec_](https://github.com/user-attachments/assets/bc52dccf-1455-4a89-ba4b-bc0c3bcb28e4)
Like the gif above, Press 1 and 2 in a short time get the purple cube,and 1 and 3 get green, is that what you want?

**Comment by UBaer21** [comment]
Yes but it turns out this is not working always.
e.g if you have parts (not objects)

**Comment by UBaer21** [comment]
These are objects not parts.
merge them into an assembly so you have parts.
![image](https://github.com/user-attachments/assets/4c04680d-d479-4a21-a234-606eaa2e11a5)

**Comment by JohnSmithCYM** [comment]
![20241126181307_rec_](https://github.com/user-attachments/assets/34e3ea50-f050-4dd4-9b9b-698079120747)

**Comment by UBaer21** [comment]
interesting - i am using the objects browser to select the parts - then it is not working.
but i can confirm your way that it works when you click the parts. 
Further it doesn't seem to be an issue with objects and parts it is clicking on the plate view works but selecting via the object list doesn't.
![image](https://github.com/user-attachments/assets/3e64699a-ec7f-4713-ab2b-1adeed42613b)

**Comment by JohnSmithCYM** [comment]
@UBaer21 I use the 'Alt+Left mouse' to select the indivisual part then press the number of filament.

**Comment by UBaer21** [comment]
@JohnSmithCYM  Yes i understood that - but for complex multimaterial/color prints clicking the objects/parts is very inconvenient compared to going through a list of 10 parts and press the number of filaments (without mouse) - which works for filament slots 1-9 but not the other. Also selecting the items via mouse clicking in the list doesn't change that. 
I also have designs where 3D view clicking is impossible as the colored parts are embedded and not visible at all in the "prepare" tab (shine through) or at least require reorientation.
